lib/utils.js

```diff
@@ -334,43 +319,30 @@ const utils = {
     return statusByFile;
   },
 
-  getShallowRemoteStatus: async (type, resources, pathTemplate) => {
+  getShallowRemoteStatus: async (localFiles, resources, pathTemplate) => {
+    if (!resources) { return {}; }
     const statusByFile = {};
-    const meta = await utils.loadLocalMeta(type) || {};
-    const metaByFile = {};
-    const metaFiles = new Set();
-    keys(meta).forEach((file) => {
-      const item = meta[file];
-      metaByFile[file] = Object.assign({}, item, { file: file });
-      metaFiles.add(file);
-    });
     const compiledTemplate = template(pathTemplate);
     resources.forEach((resource) => {
       const file = path.normalize(compiledTemplate(resource));
-      metaFiles.delete(file);
       const statObj = {
         id: resource.id,
         file: file,
-        name: resource.name
+        name: resource.name,
+        status: 'found'
       };
-      if (!metaByFile[file]) {
-        statObj.status = 'added';
-      } else {
-        statObj.status = 'modified';
-      }
       statusByFile[file] = statObj;
+      localFiles.splice(localFiles.indexOf(file, 1));
```

Comment:
Notice the very sneaky and very impactful syntax error here...

Reply:
You should probably make sure the localFile exists in the array before you try to splice it off.